### DISCUSSION TITLE
Chunked upload

### DISCUSSION THREAD

**Discussion by fellettigiacomo** [immich-app/immich#13026] [Feature Request]
I would like to implement the upload on the mobile app of chunked files, to bypass Cloudflare's proxy 100mb upload limit.
Are you guys interested in such a feature?

Thanks!

**Comment by bo0tzz** [comment]
Previous discussion: #1674

We're interested in this, but before starting on any code please discuss your planned approach with us, either here or on Discord.

**Reply by ErasmusAndre** [reply]
This is the 2nd highest requested feature and has been open since 2022, I really hope it gets some attention.

**Comment by adamzvolanek** [comment]
Hoping to see this functionality implemented myself. My immich is also behind Cloudflare and not able to upload videos without chunked upload capability.

**Reply by ErasmusAndre** [reply]
A lot of people run all their services through cloudflare, needing a special proxy setup for Immich and being able to also run it through cloudflare is a real pain.

There are also other benefits to not uploading huge files in one big chunk like when on slow or unstable connections.

**Comment by adamzvolanek** [comment]
Has there been an update to this feature or placed on the roadmap?

**Reply by CrazyTim71** [reply]
Yes, the feature is now listed on the official roadmap. You can check it on their website.

**Reply by ErasmusAndre** [reply]
Fantastic news!!

**Reply by opensourceismysecondgirlfriend** [reply]
> Yes, the feature is now listed on the official roadmap. You can check it on their website.

Which is nice, but afaik SoonTM could mean 2029. 
Looking forward to it tho!
